### DISCUSSION TITLE
[Swiper/Elements][React] Maybe I found a bug

### DISCUSSION THREAD

**Discussion by matheustodao** [nolimits4web/swiper#7018] [General]
I was upgrading `Swiper React` to `Swiper Element` and when I tried to add `navigation` or `pagination` parameters I got the following error.

![image](https://github.com/nolimits4web/swiper/assets/52856911/10c80c78-0da8-4a5b-8882-c6860f3d44a0)

According to the tests I did I get this error only when I pass `navigation` or `pagination` as false.

I made a demonstration https://github.com/matheustodao/demo-swiper-params-bug

```tsx
// src/components/swiper/container.tsx
 export function Swiper({
	children,
	swiperParams,
}: ISwiperProps) {
  const swiperRef = useRef(null);

  useEffect(() => {
    // Register Swiper web component
    register();

    // Object with parameters
    const params = {
      slidesPerView: 1,
      pagination: false,
      breakpoints: {
        768: {
          slidesPerView: 4,
        },
      },
    };

    // Assign it to swiper element
    Object.assign(swiperRef.current, params);

    // initialize swiper
    swiperRef.current.initialize();
  }, []);

  return (
    <swiper-container init="false" ref={swiperRef}>
      {children}
    </swiper-container>
  );
}
```

**Comment by nolimits4web** [comment]
false is not a valid value for pagination, use {enabled: false} instead

**Comment by chachaadmin** [comment]
I have the same problem. Same error. Anything about it?

`import React, { useRef, useEffect } from 'react';
import { __ } from '@wordpress/i18n';
import { register } from 'swiper/element/bundle';

export default function Slider(props) {

    const swiperElRef = useRef(null);

    // Retrieve all the variables and functions needed (Attributes, etc)
    const {
        attributes
    } = props;

    // Set the section id
    useEffect(() => {

        register();

        if (swiperElRef && swiperElRef.current) {
            const params = {
                navigation: { enabled: attributes.navigation.show },
                pagination: { enabled: attributes.pagination.show }
            };

            Object.assign(swiperElRef.current, params);
            swiperElRef.current.initialize();
        }
    }, [attributes]);

    return (
        <swiper-container ref={swiperElRef} init="false">
            {attributes.images.map((image, index) => {
                return (
                    <swiper-slide key={index}>
                        <img src={image?.sizes?.full?.url} width={image?.sizes?.full?.width} height={image?.sizes?.full?.height} alt={image?.alt} />
                    </swiper-slide>
                );
            })}
        </swiper-container>
    );
}`
